File: src/main/java/com/leon/bugreport/listeners/ReportListener.java
package com.leon.bugreport.listeners;

import com.leon.bugreport.BugReportManager;
import org.bukkit.Bukkit;
import org.bukkit.ChatColor;
import org.bukkit.entity.Player;
import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;
import org.bukkit.inventory.Inventory;

public class ReportListener implements Listener {
	@EventHandler
	public void onReportCreated(ReportCreatedEvent event) {
		for (Player player : Bukkit.getOnlinePlayers()) {
			if (player.getOpenInventory().getTitle().startsWith(ChatColor.YELLOW + "Bug Report")) {
				Inventory newGui = BugReportManager.generateBugReportGUI(1, false);
				player.openInventory(newGui);
			}
		}
	}
}

File: src/main/java/com/bugreportmc/bugreport/listeners/ReportListener.kt
package com.bugreportmc.bugreport.listeners

import com.bugreportmc.bugreport.BugReportManager.Companion.generateBugReportGUI
import org.bukkit.Bukkit
import org.bukkit.ChatColor
import org.bukkit.event.EventHandler
import org.bukkit.event.Listener
import org.bukkit.inventory.Inventory

class ReportListener : Listener {
	@EventHandler
	fun onReportCreated(event: ReportCreatedEvent?) {
		for (player in Bukkit.getOnlinePlayers()) {
			if (player.openInventory.title.startsWith(ChatColor.YELLOW.toString() + "Bug Report")) {
				val newGui: Inventory = generateBugReportGUI(1, player, false)
				player.openInventory(newGui)
			}
		}
	}
}
